### PR TITLE
Audio_Format_Converter

### LINKED ISSUE #453: Audio File Converter
# Description

A script that can convert various formats of audio files into other formats, preferably to ```.wav``` format. It should take an audio file, its sample rate, number of channels as an argument, and should allow the user to resample the file and convert it into ```.wav``` format.  Other basic utilities like Renaming, changing File Destination, etc are also appreciated (Not important).

## Language

- [ ] Python

## Type of issue

- [ ] Feature

## Checklist:

- **For Feature**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Audio File Converter | A script to convert an audio file into various format | Python, Hacktoberfest, SLoP | @vybhav72954 |

**Is your feature request related to a problem? Please describe.**: 
While working on Audio Feature Extraction, I noticed that  ```.wav``` files are the vest format for Audio Processing because it is more accurate than mp3, is lossless, and also size optimized (unlike ```flac```). The problem is that there are not many sites from where you can download such files, and BenchMarked Datasets like GTZAN and EmoMusic also contain only `mp3` file. A script that would auto-convert such files would be much appreciated.

**Describe the solution you'd like**: 

A simple Python script that can convert audio files along with some additional functionality like downsampling, mono to stereo conversion.

### PR DESCRIPTION
**Pull request by vybhav72954**
# Description

A script to convert various formats of audio files into other formats. It takes audio files as input, can convert them into a particular format, saves them in a particular directory, and can change the sampling rate.

Fixes #453  

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by vybhav72954** [comment]
@HarshCasper Also see this please

File: Python/Audio_Format_Converter/Audio_File_Converter.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Script to Convert audio files from one format to another
+import os
+from glob import glob
+import pydub
+import mutagen
+
+# Directory containing the audio File
+song_dir = 'C:\\Users\\asus\\Desktop\\Songs\\*.mp3'  # Extension is according the the file type before conversion.
```
Comment (HarshCasper) [inline Python/Audio_Format_Converter/Audio_File_Converter.py:8]: Remove any Local References in File Paths as per your Local Machine.

File: Python/Audio_Format_Converter/Audio_File_Converter.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                .mp4,
+                .ogg,
+                .aac,
+                .flv,
+                .mkv,
+                .flac,
+                .wav
+        """
+        sound = pydub.AudioSegment.from_mp3(song)
+        sound.export(mp3_song, format="wav")  # Change the extension to the desired format
```
Comment (HarshCasper) [inline Python/Audio_Format_Converter/Audio_File_Converter.py:25]: Can you try putting the comment above the Line of Code. You can follow the same for all other Comments as well.

File: Python/Audio_Format_Converter/Audio_File_Converter.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        os.remove(song)  # If you want to keep  the old format, comment this line
```
Comment (HarshCasper) [inline Python/Audio_Format_Converter/Audio_File_Converter.py:26]: You can possibly add an If-Else Block to ask the user if they want to keep the Old Format.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Kindly make the above mentioned changes

File: Python/Audio_Format_Converter/Audio_File_Converter.py (hunk 1 above)
Comment (vybhav72954) [reply Python/Audio_Format_Converter/Audio_File_Converter.py]: Updated in the latest commit

File: Python/Audio_Format_Converter/Audio_File_Converter.py (hunk 2 above)
Comment (vybhav72954) [reply Python/Audio_Format_Converter/Audio_File_Converter.py]: Updated in the latest commit

File: Python/Audio_Format_Converter/Audio_File_Converter.py (hunk 3 above)
Comment (vybhav72954) [reply Python/Audio_Format_Converter/Audio_File_Converter.py]: Added a keyboard interrupt feature.

**Review by vybhav72954 — COMMENTED** [review]
Done and added another small feature as well.

**Comment by vybhav72954** [comment]
@HarshCasper Thanks for the styling suggestions, can you review it again?

**Comment by vybhav72954** [comment]
@HarshCasper @srimani-programmer Can you guys review?

**Comment by vybhav72954** [comment]
@HarshCasper Waiting

**Comment by HarshCasper** [comment]
give @vybhav72954 75 points

**Pull request merged**